test/livebook_web/utils_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule LivebookWeb.UtilsTest do
+  use ExUnit.Case, async: true
+
+  alias LivebookWeb.Utils
+
+  describe "name_to_id/1" do
+    test "Empty string", do: assert(Utils.name_to_id("") == "")
+
+    test "Title case", do: assert(Utils.name_to_id("Title of a Section") == "title-of-a-section")
+
+    test "Space characters",
+      do: assert(Utils.name_to_id("  something \n  with  spaces \t  ") == "something-with-spaces")
+
+    test "Emoji at end", do: assert(Utils.name_to_id("Test 🦦 ") == "test")
+    test "Emoji in middle", do: assert(Utils.name_to_id("One 🥮 Two") == "one-two")
```

Comment:
Nit: please use regular `do end` for clarity. Also we generally use lowercased test descriptions across the project ^^

Reply:
🤷‍♂️ 😄  It's pretty clear for me, and I like the point-by-point style for each test, but I'm happy to adapt to your style 👍